python/tests/server/test_worker.py

```diff
@@ -552,10 +594,9 @@ def cancel(self, r):
 
     def teardown(self):
         self.worker.shutdown()
-        # cheat a bit to make sure we drain the events pipe
-        list(self.worker._wait())
         # really make sure everything is shut down and cleaned up
         self.worker.terminate()
+        self.await_(asyncio.sleep(0.01))
```

Comment:
Might be worth a comment to explain what this is doing.

Reply:
ended up being unnecessary